src/PerimeterxUtils.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Perimeterx;
+
+class PerimeterxUtils
+{
+	private $customParamsArray = [
+		'custom_param1' => '',
+		'custom_param2' => '',
+		'custom_param3' => '',
+		'custom_param4' => '',
+		'custom_param5' => '',
+		'custom_param6' => '',
+		'custom_param7' => '',
+		'custom_param8' => '',
+		'custom_param9' => '',
+		'custom_param10' => ''
+	];
+
+	public function handleCustomParams($pxConfig, &$array) {
+		if (isset($pxConfig['enrich_custom_params'])) {
+			$riskCustomParams = $pxConfig['enrich_custom_params']($customParamsArray);
```

Comment:
it's no longer just for risk, so the name `riskCustomParams` might be a bit confusing